contentcuration/contentcuration/static/js/edit_channel/new_channel/views.js

```diff
@@ -115,6 +140,7 @@ var ChannelListItem = BaseViews.BaseListEditableItemView.extend({
 	delete_channel: function(event){
 		if(this.isNew){
 			this.delete(true, " ");
+			this.containing_list_view.set_editing(false);
```

Comment:
https://trello.com/c/c6wzcNXn/220-disabled-new-channel-and-edit-button-after-adding-and-deleting-a-new-channel-on-the-same-page